docs/developer/tutorials/logs/submit-logs-from-http.md

```diff
@@ -0,0 +1,142 @@
+# Submit Logs from HTTP API
+
+## Getting Started
+
+This tutorial assumes you have done the following:
+
+- [set up your environment](../../index.md#getting-started)
+- read the [logs crawler documentation](../../base/logs-crawlers.md)
+- read about the [HTTP capabilities](../../base/http.md) of the base class
+
+Let's say we are building an integration for an API provided by *ACME Inc.*
+Run the following command to create the scaffolding for our integration:
+
+```
+ddev create ACME
+```
+
+This adds a folder called `acme` in our `integrations-core` folder.
+The rest of the tutorial we will spend in the `acme` folder.
+```
+cd acme
+```
+
+In order to spin up the integration in our scaffolding, if we add the following to `tests/conftest.py`:
+
+```python
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield {'tags': ['tutorial:acme']}
+```
+
+Then run:
+```
+ddev env start acme py3.11 --base
```

Comment:
Let's simplify:

```suggestion
ddev env start acme py3.11
```

Reply:
I think we need this for hot reload to work. It's either this or `--dev`.

Reply:
I'll change it to `--dev`, I just tried and that's necessary to pick up the changes.